libmamba/include/mamba/core/subdirdata.hpp

```diff
@@ -17,11 +16,12 @@
 #include "mamba/core/context.hpp"
 #include "mamba/core/download.hpp"
 #include "mamba/core/error_handling.hpp"
-#include "mamba/core/mamba_fs.hpp"
+#include "mamba/core/fetch.hpp"
```

Comment:
Note: subdirdata has been refactored and does not depend anymore on this file, so we should not include it. This can be removed later, when `fetch.hpp` is removed from the code base.